client/src/app/components/date-picker/date-picker.component.ts

```diff
@@ -3,16 +3,18 @@ import { MatDatepickerInputEvent } from '@angular/material/datepicker';
 import { FormControl } from '@angular/forms';
 import { NgbTimeStruct } from '@ng-bootstrap/ng-bootstrap';
 import { NgbTimepickerConfig } from '@ng-bootstrap/ng-bootstrap';
+import * as moment from 'moment';
 @Component({
     selector: 'app-date-picker',
     templateUrl: './date-picker.component.html',
     styleUrls: ['./date-picker.component.scss']
 })
 export class DatePickerComponent implements OnInit, OnChanges {
-    @Output() public onDatePicked: EventEmitter<Date> = new EventEmitter<Date>();
+    @Output() public onDatePicked: EventEmitter<moment.Moment> = new EventEmitter<moment.Moment>();
```

Comment:
Please return as utc (iso) string. This way you can use this component anywhere, and convert output date to JS object or moment object if needed.

Reply:
Changed. Now date-picker emits date as a string.